### DISCUSSION TITLE
可不可以加一个添加到我喜欢之后红心变成实心的

### DISCUSSION THREAD

**Discussion by KaitouKidgmh** [amll-dev/applemusic-like-lyrics#218] [General]
可不可以加一个添加到我喜欢之后红心变成实心的

**Comment by Linho1219** [comment]
AMLL bncm 插件已停止维护，最终版本存档于 <https://github.com/amll-dev/applemusic-like-lyrics-proto>。

现关闭此讨论，敬请谅解。
